pkg/detectors/rubygems/rubygems.go

```diff
@@ -47,7 +47,7 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 		}
 
 		if verify {
-			req, err := http.NewRequestWithContext(ctx, "GET", "https://rubygems.org/api/v1/gems", nil)
+			req, err := http.NewRequestWithContext(ctx, "GET", "https://rubygems.org/api/v1/gems.json", nil)
```

Comment:
Maybe we should use `GET - /api/v1/profile/me.json` instead?

It returns a body like this ```[
  {
    "id": 1,
    "handle": "qrush",
    "mfa": "enabled"
  }
]
```

We could surface `username` and `mfa_enabled` in the extra data field

Reply:
https://guides.rubygems.org/rubygems-org-api/

Reply:
It probably doesn't ever return forbidden either

Reply:
GET - /api/v1/profile/me.(json|yaml)
View basic user information for your account, including Multi-factor authentication status. Requires username and password to be passed.

it requires username and password to be passed